web/src/narrow_banner.ts

```diff
@@ -160,6 +162,23 @@ function pick_empty_narrow_banner(): NarrowBannerData {
             return default_banner;
         }
 
+        if (_.isEqual(current_terms_types, ["sender", "has-reaction"])) {
+            return {
+                title: $t({defaultMessage: "You have no messages with reactions yet!"}),
```

Comment:
This needs something more precise, like "None of your messages have emoji reactions yet." 

Absent clarity, someone could read "your messages" as the ones they've sent or the ones they've received.